### DISCUSSION TITLE
`DiscordOAuthClient` method `login` should have an option to return the token.

### DISCUSSION THREAD

**Discussion by chrisdewa** [nwunderly/starlette-discord#10] [Ideas]
An idea of what i'm suggesting:
```python
async def login(self, code, return_token=False):
    """Shorthand for session setup + identify."""
    async with self.session(code) as session:
        user = await session.identify()
        token = session.token
    return user if not return_token else user, token
```

The idea is that currently i feel that `login` method is not very useful considering the last commit, but if it's a way to get the user object AND the token in one like of code, i belive it's worth it.

**Comment by nwunderly** [comment]
Added in [v0.2.0](https://starlette-discord.nwunder.com/en/latest/api.html#starlette_discord.DiscordOAuthClient.login_return_token)!
